src/modules/tunnel/unabto_tunnel_common.c

```diff
@@ -384,3 +383,16 @@ void unabto_tunnel_epoll_event(struct epoll_event* event)
 #endif
 }
 #endif
+
+#if NABTO_ENABLE_TUNNEL_TCP && NABTO_ENABLE_TUNNEL_STATUS_CALLBACKS
+
+void unabto_tunnel_status_get_tcp_info(tunnel* tunnel, tunnel_status_tcp_details* info) {
```

Comment:
this function should only return data if the tunnel type is tcp. Else the callback should be tcp specific by name. Such that the caller has a good idea that the tunnel* is a tcp tunnel